### ISSUE TITLE
Validate provider URL templates

### ISSUE THREAD

**Issue by mojodna** [fieldpapers/fieldpapers#41]
Ensure that they include appropriate placeholder values.

**Comment by sconnelley** [comment]
Is this custom URL templates?

**Comment by mojodna** [comment]
Yes, but it can safely apply to all, since that's how we're passing around ones from the default list.

This would go as a validation in `atlas.rb`

**Comment by sconnelley** [comment]
I'm validating template strings in #40 via Javascript using this regex
```
/https?:\/\/(\{[s]\})?[\/\w\.\-\?\+\*_\|~:\[\]@#!\$'\(\),=&]*\{[zxy]\}\/\{[zxy]\}\/\{[zxy]\}\.(jpg|png)$/gi
```
Would this satisfy the requirement?

**Comment by mojodna** [comment]
We should probably also validate the format (http://guides.rubyonrails.org/active_record_validations.html#format) of the `provider` property in `Atlas` on the server-side to facilitate inputs that bypass JS.  This regex is a great start and should be able to be dropped in.

**Comment by sconnelley** [comment]
Also, I guess since `Atlas provider` can look like this `http://{s}.basemaps.cartocdn.com/light_nolabels/{z}/{x}/{y}.pnghttp://a.tiles.mapbox.com/v3/americanredcross.HAIYAN_Atlas_Bounds/{Z}/{X}/{Y}.pnghttp://tile.stamen.com/utm/{Z}/{X}/{Y}.png` we would need to split this up before validating
